styleguide/screens/Design/Responsive.js

```diff
@@ -0,0 +1,144 @@
+    </T>
+    <T elm="i">
+      John Allsop, "<A href="http://www.alistapart.com/articles/dao/">A Doa of Web Design</A>"
+    </T>
+    <D>
+      <H level={ 2 }>Bottom up</H>
+      <T elm="p" className={ m.mtr }>
+        When it comes to design and building out our interfaces we take a bottom up approach.
+        By starting to consider a page in the smallest context it will appear, such as a device
+        with a small screen, we are forced to prioritise the important parts of the page. This
+        allows us to ensure everything we add to the page provides value to the user, and forces
+        us to be ruthless in removing unnecessary cruft and excess features.
+      </T>
+      <T elm="p" className={ m.mtr }>
+        By looking at the smallest context first, we provided feature parity across all devices,
+        and provides users of mobile and touch devices with a first class experience.
+      </T>
+    </D>
+    <D>
+      <H level={ 2 }>Media queries</H>
+      <T elm="p" className={ m.mtr }>
+        Media queries allow us to apply CSS rules depending on characteristics of a device or
+        browser window. They allow us to create responsive web pages, which are suitable for
+        use across many different devices.
+      </T>
+      <T elm="p" className={ m.mtr }>
+        As described <A href="#">above</A> you should take a bottom up approach.
+        This means, writing your CSS to address the smallest sizes first.
+        We do this for two key reasons:
+      </T>
+      <ol className={ m.mtr }>
+        <T elm="li">
+          Larger viewport layouts are typically more complex than their small counterparts.
+          With this in mind, you’ll likely end up writing fewer CSS rules to achieve the desired
+          result across all scenarios.
+        </T>
+        <T elm="li">
+          By building the page for the lowest common denominator, you’ll be taking into account
+          devices that we’re unable to test on, or even those that don’t exist yet.
+        </T>
+      </ol>
+      <T elm="p" className={ m.mtr }>
+        When working “bottom up”, it should apparent become that you need to add a media query when
+        the page or feature begins to look and feel rubbish. Look to increase the window size until
+        this point, add a media query which adds additional styles which improve the page or
+        feature’s appearance, then rinse and repeat. Use common sense with doing this, as we don’t
+        want to add too many media queries at the expense of bloat in our code.
+      </T>
+      <Specimen
+        classNames={ {
+          root: m.mtr,
+          specimenContainer: m.par,
+        } }
+        code={ dedent`
+          .container {
+            display: inline-block;
+            width: 100%;
+          }
+
+          @media(min-width: 30rem) {
+            .container {
+              width: 50%;
+            }
+          }
+        ` }
+      >
+        <H level={ 3 }>Example</H>
```

Comment:
This isn't so great. Why don't we just have a code block component so we don't need to unnecessarily fill the Specimen's child prop? If not, how come you're using a `h3` here?